tests/foreman/api/test_contentview.py

```diff
@@ -1188,7 +1190,7 @@ def test_positive_admin_user_actions(content_view, function_role, module_org, mo
         login=user_login,
         password=user_password,
     ).create()
-    cfg = get_nailgun_config()
+    cfg = deepcopy(default_sat.nailgun_cfg)
```

Comment:
same comment about this not existing until api is poked

Reply:
Moved this commit, scope will get too big.